packages/tupaia-web-server/src/routes/ProjectCountryAccessListRoute.ts

```diff
@@ -43,10 +42,10 @@ export class ProjectCountryAccessListRoute extends Route<ProjectCountryAccessLis
         return [
           ...result,
           {
-            id: country?.id,
+            id: country.id,
             name,
             hasAccess,
-            hasPendingAccess,
+            hasPendingAccess: country.hasPendingAccess,
```

Comment:
Is this endpoint doing the same as the central server endpoint now? If so, we can probably remove this and forward to central server

Reply:
Yep! Equivalent behaviour. Just pushed an update to retire this one and just forward to central-server